lsf_client/lsf_client.py

```diff
@@ -7,10 +7,14 @@ class LSFClient(object):
 
     def submit(self, command, stdout):
         bsub_command = ['bsub', '-sla', settings.LSF_SLA, '-oo', stdout]
+        toil_lsf_args = '-sla %s' % settings.LSF_SLA
         if settings.LSF_WALLTIME:
             bsub_command.extend(['-W', settings.LSF_WALLTIME])
+            toil_lsf_args = '%s -W %s' % (toil_lsf_args,settings.LSF_WALLTIME)
```

Comment:
I think this would fail pep8 validation. `toil_lsf_args = '%s -W %s' % (toil_lsf_args,settings.LSF_WALLTIME)`. Missing blank space, It should be `toil_lsf_args = '%s -W %s' % (toil_lsf_args, settings.LSF_WALLTIME)`. Let's try to write pep8 valid code for new stuff so it is easier for us when we include pep8 into tests.

Reply:
resolved with #fe1df2a